src/modules/comment.js

```diff
@@ -1,16 +1,32 @@
+const popUp = document.querySelector('.comment-popup');
 export default class Comment {
   static createCommentPopUp = (data) => {
     const newArticle = document.createElement('article');
     newArticle.innerHTML = `<div class= "flex"> 
-  <img src= ${data.url} alt="Image"/> 
+  <img src= ${data.artworkUrl100} alt="Image"/> 
   <span class='material-symbols-outlined'>close</span>
   </div>
-  <h2>Space ${data.count}</h2>
+  <h2>Space ${data.index}</h2>
```

Comment:
- [ ] I think we could use the name of the album here.  
![image](https://user-images.githubusercontent.com/55507343/169337388-a699a636-e8cd-4354-bcbb-6046774673cc.png)